deploy_terraform_eks/action.yml

```diff
@@ -103,6 +109,8 @@ runs:
         cd terraform/deployment
         terraform init -backend-config=${{ inputs.service_stage }}/${{ inputs.service_stage }}_${{ inputs.aws_region }}_backend_config.tfvars 
         terraform apply -auto-approve -var-file=${{ inputs.service_stage }}/${{ inputs.service_stage }}_${{ inputs.aws_region }}.tfvars \
+                                      -var "route53_access_key=${{ inputs.route53_access_key }}" \
```

Comment:
this is going to break all other deployments that don't use route53 at all. You should either switch it to make it optional and do some logic to only pass the variables if they are present OR create/use another action (we have the migrate_terraform_eks that have this)